check-version.sh

```diff
@@ -51,11 +51,14 @@ if [ -z $checksum_tool ]; then
     exit 4
 fi
 
-mkdir -p /tmp/$USER
-tmpfile=$(mktemp /tmp/$USER/XXXXX)
-
-find $dir -name '*.tex' | sort -u | xargs $checksum_tool > $tmpfile
-new_checksum=$($checksum_tool $tmpfile | cut -f1 -d' ')
+function compute_checksum() {
+    tmpfile=$(mktemp /tmp/${USER}_XXXXX)
```

Comment:
why hardcode the path? Let `mktemp` handle it completely? What if the user has a different `$TMPDIR` variable?